diff --git a/node_modules/react-native/Libraries/Components/Keyboard/Keyboard.js b/node_modules/react-native/Libraries/Components/Keyboard/Keyboard.js
index fec163c..a009288 100644
--- a/node_modules/react-native/Libraries/Components/Keyboard/Keyboard.js
+++ b/node_modules/react-native/Libraries/Components/Keyboard/Keyboard.js
@@ -143,7 +143,7 @@ class Keyboard {
   /**
    * @deprecated Use `remove` on the EventSubscription from `addEventListener`.
    */
-  removeEventListener<K: $Keys<KeyboardEventDefinitions>>(
+  removeListener<K: $Keys<KeyboardEventDefinitions>>(
     eventType: K,
     listener: (...$ElementType<KeyboardEventDefinitions, K>) => mixed,
   ): void {
diff --git a/node_modules/react-native/scripts/.packager.bat b/node_modules/react-native/scripts/.packager.bat
new file mode 100644
index 0000000..e89260e
--- /dev/null
+++ b/node_modules/react-native/scripts/.packager.bat
@@ -0,0 +1 @@
+set RCT_METRO_PORT=8081
\ No newline at end of file
